examples/material_gallery/lib/gallery/app.dart

```diff
@@ -5,8 +5,45 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/scheduler.dart' show timeDilation;
 
+import '../demos.dart';
 import 'home.dart';
 
+final Map<String, WidgetBuilder> kRoutes = <String, WidgetBuilder>{
+  WeatherDemo.routeName: (_) => new WeatherDemo(),
```

Comment:
We should put in the types, because people copy/paste this code and they need an easy way to find out what type they're getting.